packages/integrations/image/README.md

```diff
@@ -126,7 +126,7 @@ For remote images, provide the full URL. (e.g. `src="https://astro.build/assets/
 
 Defines an alternative text description of the image.
 
-Set to an empty string (alt="") if the image is not a key part of the content (it's decoration or a tracking pixel).
+Set to an empty string (`alt=""`) if the image is not a key part of the content (it's decoration or a tracking pixel).
```

Comment:
```suggestion
Set to an empty string (`alt=""`) if the image is not a key part of the content (e.g. it's decoration or a tracking pixel).
```
@delucis - This exact sentence also occurs for the `<Picture />` component, so you'll need to do the same wrapping around `alt=""` (and pls add "e.g." there, too).